feature/circle-align

jufa2401 opened the pull request:
To be tested...

jufa2401 committed:
align method pushed changed to switch case for readability and fewer lines of code.

jufa2401 committed:
Merge remote-tracking branch 'origin/master' into feature/align-circle

# Conflicts:
#	autonomous-drone/src/main/java/dk/localghost/hold17/autonomous_drone/controller/KeyboardCommandManager.java

jufa2401 committed:
merged with latest master

jufa2401 committed:
Merge remote-tracking branch 'origin/master' into feature/align-circle

jufa2401 committed:
added spacing. Changed names to comply the Troels

jufa2401 committed:
corrected method call, outcommented unused method.

jufa2401 committed:
corrected method call and added the correct method for finding the Direction

jufa2401 committed:
refactor and codestyle update

jufa2401 committed:
Merge remote-tracking branch 'origin/feature/codestyle-and-refactor' into feature/align-circle

# Conflicts:
#	autonomous-drone/src/main/java/dk/localghost/hold17/autonomous_drone/opencv_processing/Direction.java
#	autonomous-drone/src/main/java/dk/localghost/hold17/autonomous_drone/opencv_processing/filter/Benchmark.java

jufa2401 committed:
recommit

tjoels commented:
Jeg tror ikke at 9-segment inddelingen kommer til at blive brugt. Den skal hellere reduceres til 3-segment med kun LEFT, CENTER og RIGHT.
Switch cases i DroneController bør ikke indeholde cases til LEFTUP, LEFTDOWN etc.
Hvis det skulle være konsekvent, burde LEFTCENTER også være håndteret.

jufa2401 committed:
Merge remote-tracking branch 'origin/master' into feature/align-circle

# Conflicts:
#	autonomous-drone/src/main/java/dk/localghost/hold17/autonomous_drone/controller/DroneController.java
#	autonomous-drone/src/main/java/dk/localghost/hold17/autonomous_drone/gui/GUIController.java
#	autonomous-drone/src/main/java/dk/localghost/hold17/autonomous_drone/opencv_processing/CircleFilter.java
#	autonomous-drone/src/main/java/dk/localghost/hold17/autonomous_drone/opencv_processing/filter/Benchmark.java
#	autonomous-drone/src/main/java/dk/localghost/hold17/autonomous_drone/opencv_processing/util/PointComparator.java

jufa2401 committed:
fixed enum and changed things

jufa2401 committed:
fixed QRCodeScanner by moving it to a Timertask
Runs circlerecognition is timertask on start.
Updated direction enum and more...

jufa2401 committed:
Added colors to certain string outputs

jufa2401 committed:
added class for universal use, could not find a package at a higher level. For correct use see commandmanager send command

jufa2401 committed:
Colored more sout calls.

jufa2401 committed:
added another color

jufa2401 committed:
added another color

twast92 committed:
Update DroneController.java

Use MAX_ALTITUDE

twast92 committed:
Update GUIController.java

Removed print image width as this will be done inside the decoder in a future commit
